libs/fabric/src/lib/components/combo-box/directives/combo-box-option.directive.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { Directive, Input } from '@angular/core';
+import { IComboBoxOption } from 'office-ui-fabric-react';
+
+/**
+ * Wrapper directive for creating a ComboBoxOption
+ */
+@Directive({ selector: 'fab-combo-option' })
```

Comment:
How about renaming to `fab-combo-box-option` to stay consistant with the component's name?

Reply:
Done.